fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -3221,8 +3236,8 @@ public int doReindex(FHIROperationContext operationContext, OperationOutcome.Bui
      *            the resource type against which the interaction is to be performed
      * @throws FHIROperationException
      */
-    private void validateInteraction(String interaction, String resourceType) throws FHIROperationException {
-        List<String> interactions = null;
+    private void validateInteraction(Interaction interaction, String resourceType) throws FHIROperationException {
```

Comment:
I originally made this update when I was thinking to check the interaction and resource type here.  It proved unnecessary, but I ended up leaving it because I prefer passing around the enum and only converting to strings when we need to.